performance-tools/performance-cluster/scripts/helpers/k6-runner.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Expected vars:
+# - INFLUXDB_FQDN: FQDN for influxDB
+# - TESTS_DIR: Directory where all the tests are located
+
+set -o errexit
+set -o pipefail
+
+
+# Setup
+export K6_USER="$(cat /var/k6-details/k6admin)"
+export K6_PASSWORD="$(cat /var/k6-details/k6admin_pass)"
+export INFLUXDB="$(cat /var/k6-details/k6database)"
+
+TESTS_DIR='./tests'
+
+function runAll() {
+  shout "Running the complete suite"
```

Comment:
Formatting is inconsistent with the other scripts of the PR. Here you use 2 spaces for indentation and above I see 4 spaces used for everything else. Can we use consistent formatting?